notebooks/feature_engineering/solutions/4_keras_adv_feat_eng.ipynb

```diff
@@ -31,15 +31,23 @@
    "metadata": {},
```

Comment:
<p>Remove the learning objective "Process temporal feature columns in Keras" since we no longer do that in the lab.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>+1</p>